src/AccessibilityInsights.SharedUx/Controls/EventConfigurationControl.xaml

```diff
@@ -41,7 +41,7 @@
         </Grid.RowDefinitions>
         <Border Grid.Row="0" BorderThickness="1" BorderBrush="LightGray">
             <DockPanel>
-                <fabric:FabricIconControl GlyphName="Search" Foreground="{DynamicResource ResourceKey=IconBrush}" FontSize="18" GlyphSize="Custom" Margin="5,0,5,0" DockPanel.Dock="Left" VerticalAlignment="Center" HorizontalAlignment="Center" />
+                <fabric:FabricIconControl GlyphName="Search" Foreground="{DynamicResource ResourceKey=IconBrush}" FontSize="18" GlyphSize="Custom" Margin="5,0,5,0" DockPanel.Dock="Left" VerticalAlignment="Center" HorizontalAlignment="Center" ShowInControlView="False"/>
```

Comment:
![search icon in events config](https://user-images.githubusercontent.com/7775527/64988693-e28fd000-d880-11e9-9dbd-16ed41bbe5ad.png)

Reply:
I'm not sure this particular icon is non-decorative because I think it's adding new information for a visual user; I don't think I'd have had a good guess about what the box was for without the presence of the search icon.

Reply:
moved back to control view for now